releasenotes/notes/upgrade-qpy-schedule-f28f6a48a3abb4de.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    QPY serialization has been upgraded to support :class:`~qiskit.pulse.schedule.ScheduleBlock`.
+    The import path `qiskit.circuit.qpy_serialization` has been deprecated and will be removed in
+    a future release. QPY version has been bumped to 3 due to the data structure update.
+    Loading QPY data generated with the previous versions is still supported.
```

Comment:
We should have release notes here for all the deprecations both in the pulse alignment stuff and then the named kwarg `circuits` for the interface functions